### DISCUSSION TITLE
add support for container

### DISCUSSION THREAD

**Discussion by DBTK1990** [orta/vscode-jest#879] [Ideas]
i got it working with the following npm command  ` docker run --rm --network self-schedule-db_db-network -v $INIT_CWD:/home/node/server -v $HOME/.vscode-server/extensions/orta.vscode-jest-4.6.0:$HOME/.vscode-server/extensions/orta.vscode-jest-4.6.0 -v $INIT_CWD/docker/db.env:/home/node/server/.env self-schedule-main_server npm run test --`
but I want it to be more general, so I can move between our team mates without needing to get to the 4.6.0 prefix
for that I need a way to control the reporter script, maybe with overriding config

checked the code too and i think it is possible
any suggestion?

**Comment by sagsag1324** [comment]
Amazing!

**Comment by Naapperas** [comment]
Any news on this one?
